frontend/src/components/Center.test.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+  This file is part of Edgehog.
+
+  Copyright 2021-2023 SECO Mind Srl
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  SPDX-License-Identifier: Apache-2.0
+*/
+
+import { it, expect } from "vitest";
+import { render } from "@testing-library/react";
+
+import Center from "./Center";
+
+it("renders with the passed className", () => {
+  const { container } = render(
+    <Center children="innerText" className="test-class" />,
+  );
+  expect(container.firstChild).toHaveClass(
+    "h-100 d-flex flex-column justify-content-center align-items-center test-class",
+  );
+});
```

Comment:
We do not need to test actual implementation of `Center` component, I think it's better test that passing `foo-class` in `className` property this class is present in rendered container, not to check for every class. 
For example:

```suggestion
it("renders with the passed className", () => {
  const { container } = render(
    <Center children="innerText" className="foo-class" />,
  );
  expect(container.querySelector(".foo-class")).toBeInTheDocument();
});
```